stardis/plasma/base.py

```diff
@@ -301,8 +309,8 @@ def calculate(
         linelist["level_energy_upper"] = ((linelist["e_up"].values * u.eV).cgs).value
 
         # Radiation broadening parameter is approximated as the einstein A coefficient. Vald parameters are in log scale.
-        linelist["A_ul"] = 10 ** (
-            linelist["rad"]
+        linelist["A_ul"] = 10 ** (linelist["rad"]) / (
+            4 * np.pi
         )  # see 1995A&AS..112..525P for appropriate units - may be off by a factor of 4pi
```

Comment:
Is this comment still correct?

Reply:
Yes in that I'm still not quite sure, but this is consistent with how we do it in other parts of the code. It's a small effect, so hard to test and validate in the code itself. The units are (4 pi s) ^-1, and whether those units are consistent with how people apply radiation broadening or not is what's unclear to me.